README.md

```diff
@@ -145,21 +142,21 @@ This readme gives some basic examples; for our full API documentation, please go
 
 All examples assume a client has been created as follows:
 
-```javascript
+```ts
```

Comment:
It does bother me that you're needing to use `ts` highlighting for code that is definitely not TypeScript. My preference would be go go with `js` hightlighting, preferring correctness in the markdown at source level, rather than compensating for GitHub's rendering.

It looks like that may apply elsewhere in this PR, also.

Related, perhaps it's worth explicitly stating in the surrounding copy the ES level of the JavaScript being quoted?

Reply:
Unless I've made a mistake somewhere the code here should be TypeScript? It was certainly neither valid TypeScript nor valid JavaScript before. With regards to the ES level it might be best to just change the `var`s here to `const`s since no sane TypeScript developer would be using `var` but perhaps the same could be said for JavaScript code snippets both here and in the docs repo as well...